test/tests/executioners/nl_pingpong/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  [./nl_pingpong]
+    type = RunApp
+    input = 'nonlinear_residual_pingpong.i'
+    expect_out = "Nonlinear solve did not converge due to DIVERGED_LINE_SEARCH"
```

Comment:
I think I would remove ` due to DIVERGED_LINE_SEARCH` in case sometime in the future we add a `SNES_DIVERGED_USER` or something like that